style.css

```diff
@@ -0,0 +1,86 @@
+
+body html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+/* Header */
+header {
+  display: flex;
+  max-width: 500px;
+  margin: 0 auto;
+  justify-content: space-around;
+  align-items: center;
+}
+
+header h2 {
+  color: #6070ff;
+  font-size: 1.5rem;
+  padding: 0 40px;
+}
+
+nav {
+  margin: 0 auto;
+}
```

Comment:
- [ ] Kindly adjust the spacing of the hamburger menu to slightly go to the right. It's pushing too far from the right and that's not matching the original Figma design given. Please fix it 👍 

**See your design for the hamburger menu at the top**
![image](https://user-images.githubusercontent.com/50941074/170673665-74f38b11-bf95-4a9e-8878-26eaa130396b.png)

**The original Figma design**
![image](https://user-images.githubusercontent.com/50941074/170674373-3fe36912-349e-49b7-ac0c-67373c635fbb.png)

Reply:
Done